src/Sentry.Unity/Il2CppEventProcessor.cs

```diff
@@ -72,6 +72,14 @@ public void Process(Exception incomingException, SentryEvent sentryEvent)
                 //      Wouldn't it cause invalid frame info?
                 var nativeLen = nativeStackTrace.Frames.Length;
                 var len = Math.Min(sentryStacktrace.Frames.Count, nativeLen);
+
+                if (nativeLen != len)
+                {
+                    _options.DiagnosticLogger?.LogWarning(
+                        "Native and sentry stack trace lenghs don't match ({0} != {1}) - this may cause invalid stack traces.",
```

Comment:
```suggestion
                        "Native and sentry stack trace lengths don't match '({0} != {1})' - this may cause invalid stack traces.",
```